src/Models/TerminusModel.php

```diff
@@ -82,25 +85,21 @@ public function get($attribute)
     }
 
     /**
-     * Checks whether the model has an attribute
-     *
-     * @param string $attribute Name of the attribute key
-     * @return boolean True if attribute exists, false otherwise
-     */
-    public function has($attribute)
-    {
-        return isset($this->attributes->$attribute);
-    }
-
-    /**
-     * Sets an attribute
+     * Retrieves attribute of the given name and formats it according to the TERMINUS_DATE_FORMAT setting
      *
-     * @param string $attribute Name of the attribute key
-     * @param mixed $value The value to assign to the attribute
+     * @param string $attribute Name of the key of the desired date attribute
+     * @return string|null The date-formatted attribute, or null if not set.
      */
-    public function set($attribute, $value)
+    public function getDatetime($attribute)
     {
-        $this->attributes->$attribute = $value;
+        $value = $this->get($attribute);
+        if (is_null($value)) {
+            return null;
+        }
+        if (!is_integer($value)) {
```

Comment:
This actually isn't working right. The datetime fields that are formatted as epochs are still considered by php to be strings:
```
php > $s="1464574086.1292";
php > print $s;
1464574086.1292
php > var_export(is_integer($s));
false
php > var_export(strtotime($s));
false
```
This causes all of the times in workflow:list to come out as `1970-01-01 00:00:00`